### ISSUE TITLE
Umbrella issue: fix broken links in en docs

### ISSUE THREAD

**Issue by windsonsea** [DaoCloud/DaoCloud-docs#2084]
When running `mkdocs serve` in `docs/en`, the following links are broken:

- @windsonsea 

**Batch 1** #2091 
```
WARNING  -  A relative path to 'insight/dashboard/import-dashboard.md' is included in the 'nav' configuration, which is not found in the documentation files
WARNING  -  A relative path to 'storage/hwameistor/resources/volume-ProvisionedIO.md' is included in the 'nav' configuration, which is not found in the documentation files
WARNING  -  A relative path to 'kangaroo/hosted/intro.md' is included in the 'nav' configuration, which is not found in the documentation files
WARNING  -  Documentation file 'dce/dce-rn/20230531.md' contains a link to 'dce/insight/quickstart/upgrade-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'dce/dce-rn/20230630.md' contains a link to 'network/modules/spiderpool/releasenotes.md' which is not found in the documentation files.
WARNING  -  Documentation file 'dce/dce-rn/20230630.md' contains a link to 'dce/insight/quickstart/upgrade-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'dce/dce-rn/20230731.md' contains a link to 'virtnest/intro/release-notes.md' which is not found in the documentation files.
WARNING  -  Documentation file 'download/modules/insight.md' contains a link to 'insight/intro/release-notes.md' which is not found in the documentation files.
WARNING  -  Documentation file 'download/modules/kcollie.md' contains a link to 'kcollie/intro/release-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'download/modules/kcoral.md' contains a link to 'kcoral/intro/release-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'download/modules/kpanda.md' contains a link to 'kpanda/intro/release-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/intro/releasenote.md' contains a link to 'insight/quickstart/upgrade-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/agent-status.md' contains a link to 'best-practice/prometheus-res.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/agent-status.md' contains a link to 'best-practice/vms-res-plan.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/agent-status.md' contains a link to 'best-practice/modify-vms-disk.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/gethosturl.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/install-agent.md' contains a link to 'insight/quickstart/install/install/gethosturl.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/quickstart-catelogy.md' contains a link to 'insight/quickstart/install/install/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/quickstart-catelogy.md' contains a link to 'insight/quickstart/compati-test/k8s-compatibility.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/quickstart-catelogy.md' contains a link to 'insight/quickstart/compati-test/ocp-compatibility.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/quickstart-catelogy.md' contains a link to 'insight/quickstart/compati-test/rancher-compatibility.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/otel/operator.md' contains a link to 'insight/data-query/trace.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/res-plan/modify-vms-disk.md' contains a link to 'insight/best-practice/vms-res-plan.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/res-plan/vms-res-plan.md' contains a link to 'insight/quickstart/quickstart/res-plan/prometheus-res.md' which is not found in the documentation files.
```
**Batch 2** #2096 
```
WARNING  -  Documentation file 'insight/user-guide/collection-manag/collection-manag.md' contains a link to 'insight/images/collectmanage01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/collection-manag/collection-manag.md' contains a link to 'insight/images/collectmanage02.png' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/data-query/log.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/data-query/metric.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/scenario-insight/cluster.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/scenario-insight/node.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/scenario-insight/service.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'install/evolve.md' contains a link to 'install/commercial/images/succeed01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'install/import.md' contains a link to 'kpanda/best-practice/create-ubuntu-on-centos-platform.md' which is not found in the documentation files.
WARNING  -  Documentation file 'install/import.md' contains a link to 'kpanda/best-practice/multi-arch.md' which is not found in the documentation files.
```
**Batch 3** #2104 
```
WARNING  -  Documentation file 'kangaroo/integrate/bind-to-ws.md' contains a link to 'kangaroo/images/bind01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/integrate/bind-to-ws.md' contains a link to 'kangaroo/images/bind02.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/integrate/integrate-ws.md' contains a link to 'kangaroo/images/integrated01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/integrate/integrate-ws.md' contains a link to 'kangaroo/images/integrated02.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/integrate/integrate-ws.md' contains a link to 'kangaroo/images/integrated03.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/quickstart/registry-in-one.md' contains a link to 'kangaroo/images/interg05.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/space/index.md' contains a link to 'kangaroo/images/space02.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/space/reclaim.md' contains a link to 'kangaroo/images/space01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/application/tidb.md' contains a link to 'storage/hwameistor/application/file/TiDBonHwameiStor.docx' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/intro/index.md' contains a link to 'storage/dce/license0.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/intro/ls.md' contains a link to 'storage/hwameistor/hwameistor/install/deploy.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/intro/ls.md' contains a link to 'storage/hwameistor/hwameistor/install/deploy.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/intro/scheduler.md' contains a link to 'storage/hwameistor/hwameistor/install/deploy.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/ceph-dashboard.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/openebs.md' contains a link to 'storage/images/openebs-1.svg' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/openebs.md' contains a link to 'storage/images/openebs-2.svg' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/openebs.md' contains a link to 'storage/images/openebs-3.svg' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/openebs.md' contains a link to 'storage/images/openebs-4.svg' which is not found in the documentation files.
WARNING  -  Documentation file 'videos/insight.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'videos/insight.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
```
- @Michelle951 #2107 
```
WARNING  -  Documentation file 'kairship/instance/add.md' contains a link to 'kairship/instance/kpanda/intro/index.md)' which is not found in the documentation files.
WARNING  -  Documentation file 'kpanda/user-guide/security/audit.md' contains a link to 'kpanda/user-guide/kpanda/intro/index.md' which is not found in the documentation files.
WARNING  -  Documentation file 'kpanda/user-guide/security/hunter.md' contains a link to 'kpanda/user-guide/kpanda/intro/index.md' which is not found in the documentation files.
WARNING  -  Documentation file 'kpanda/user-guide/security/cis/policy.md' contains a link to 'kpanda/user-guide/kpanda/intro/index.md' which is not found in the documentation files.
WARNING  -  Documentation file 'middleware/minio/user-guide/insight.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'middleware/redis/best-practice/disaster-tolerance.md' contains a link to 'middleware/redis/images/tole00.png' which is not found in the documentation files.
WARNING  -  Documentation file 'middleware/redis/best-practice/disaster-tolerance.md' contains a link to 'middleware/redis/images/tole01.jpg' which is not found in the documentation files.
WARNING  -  Documentation file 'middleware/redis/best-practice/disaster-tolerance.md' contains a link to 'middleware/redis/images/tole02.jpg' which is not found in the documentation files.
WARNING  -  Documentation file 'skoala/gateway/logs/inslog.md' contains a link to 'insight/06UserGuide/04dataquery/logquery.md' which is not found in the documentation files.
WARNING  -  Documentation file 'skoala/quickstart/install.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'skoala/quickstart/install.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'skoala/standard/nacos.md' contains a link to 'skoala/images/standard01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'skoala/standard/sentinel.md' contains a link to 'skoala/plugins/sentinel.md' which is not found in the documentation files.
WARNING  -  Documentation file 'skoala/standard/sentinel.md' contains a link to 'skoala/monitor/components.md' which is not found in the documentation files.
WARNING  -  Documentation file 'skoala/tests/nacos-stress-test.md' contains a link to '../../zh/docs/skoala/images/test-nacos01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'skoala/trad-ms/integrated/integrate-registry.md' contains a link to 'skoala/trad-ms/hosted/registry-lcm/create-registry.md' which is not found in the documentation files.
WARNING  -  Documentation file 'videos/amamba.md' contains a link to 'amamba/user-guide/pipeline/create/custom.md' which is not found in the documentation files.
```
- @Jeanine-tw 
```
WARNING  -  Documentation file 'native/news/news-2022.md' contains a link to 'native/download/index.md' which is not found in the documentation files.
WARNING  -  Documentation file 'native/news/news-2022.md' contains a link to 'native/install/index.md' which is not found in the documentation files.
WARNING  -  Documentation file 'native/news/news-2022.md' contains a link to 'native/dce/license0.md' which is not found in the documentation files.
WARNING  -  Documentation file 'native/news/news-2023.md' contains a link to 'native/download/index.md' which is not found in the documentation files.
WARNING  -  Documentation file 'native/news/news-2023.md' contains a link to 'native/install/index.md' which is not found in the documentation files.
WARNING  -  Documentation file 'native/news/news-2023.md' contains a link to 'native/dce/license0.md' which is not found in the documentation files.
```

**Comment by windsonsea** [comment]
Fixed on Auguest 10:

```
WARNING  -  A relative path to 'insight/dashboard/import-dashboard.md' is included in the 'nav' configuration, which is not found in the documentation files
WARNING  -  A relative path to 'storage/hwameistor/resources/volume-ProvisionedIO.md' is included in the 'nav' configuration, which is not found in the documentation files
WARNING  -  A relative path to 'kangaroo/hosted/intro.md' is included in the 'nav' configuration, which is not found in the documentation files
WARNING  -  Documentation file 'dce/dce-rn/20230531.md' contains a link to 'dce/insight/quickstart/upgrade-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'dce/dce-rn/20230630.md' contains a link to 'network/modules/spiderpool/releasenotes.md' which is not found in the documentation files.
WARNING  -  Documentation file 'dce/dce-rn/20230630.md' contains a link to 'dce/insight/quickstart/upgrade-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'dce/dce-rn/20230731.md' contains a link to 'virtnest/intro/release-notes.md' which is not found in the documentation files.
WARNING  -  Documentation file 'download/modules/insight.md' contains a link to 'insight/intro/release-notes.md' which is not found in the documentation files.
WARNING  -  Documentation file 'download/modules/kcollie.md' contains a link to 'kcollie/intro/release-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'download/modules/kcoral.md' contains a link to 'kcoral/intro/release-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'download/modules/kpanda.md' contains a link to 'kpanda/intro/release-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/intro/releasenote.md' contains a link to 'insight/quickstart/upgrade-note.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/agent-status.md' contains a link to 'best-practice/prometheus-res.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/agent-status.md' contains a link to 'best-practice/vms-res-plan.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/agent-status.md' contains a link to 'best-practice/modify-vms-disk.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/gethosturl.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/install-agent.md' contains a link to 'insight/quickstart/install/install/gethosturl.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/quickstart-catelogy.md' contains a link to 'insight/quickstart/install/install/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/quickstart-catelogy.md' contains a link to 'insight/quickstart/compati-test/k8s-compatibility.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/quickstart-catelogy.md' contains a link to 'insight/quickstart/compati-test/ocp-compatibility.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/install/quickstart-catelogy.md' contains a link to 'insight/quickstart/compati-test/rancher-compatibility.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/otel/operator.md' contains a link to 'insight/data-query/trace.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/quickstart/res-plan/modify-vms-disk.md' contains a link to 'insight/best-practice/vms-res-plan.md' which is not found in the documentation files.
```

**Comment by windsonsea** [comment]
Fixed on August 10 in batch 2

```
WARNING  -  Documentation file 'insight/user-guide/collection-manag/collection-manag.md' contains a link to 'insight/images/collectmanage01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/collection-manag/collection-manag.md' contains a link to 'insight/images/collectmanage02.png' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/data-query/log.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/data-query/metric.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/scenario-insight/cluster.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/scenario-insight/node.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'insight/user-guide/scenario-insight/service.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'install/evolve.md' contains a link to 'install/commercial/images/succeed01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'install/import.md' contains a link to 'kpanda/best-practice/create-ubuntu-on-centos-platform.md' which is not found in the documentation files.
WARNING  -  Documentation file 'install/import.md' contains a link to 'kpanda/best-practice/multi-arch.md' which is not found in the documentation files.
```

**Comment by windsonsea** [comment]
Fixed batch 3 in #2104

```
WARNING  -  Documentation file 'kangaroo/integrate/bind-to-ws.md' contains a link to 'kangaroo/images/bind01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/integrate/bind-to-ws.md' contains a link to 'kangaroo/images/bind02.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/integrate/integrate-ws.md' contains a link to 'kangaroo/images/integrated01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/integrate/integrate-ws.md' contains a link to 'kangaroo/images/integrated02.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/integrate/integrate-ws.md' contains a link to 'kangaroo/images/integrated03.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/quickstart/registry-in-one.md' contains a link to 'kangaroo/images/interg05.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/space/index.md' contains a link to 'kangaroo/images/space02.png' which is not found in the documentation files.
WARNING  -  Documentation file 'kangaroo/space/reclaim.md' contains a link to 'kangaroo/images/space01.png' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/application/tidb.md' contains a link to 'storage/hwameistor/application/file/TiDBonHwameiStor.docx' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/intro/index.md' contains a link to 'storage/dce/license0.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/intro/ls.md' contains a link to 'storage/hwameistor/hwameistor/install/deploy.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/intro/ls.md' contains a link to 'storage/hwameistor/hwameistor/install/deploy.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/hwameistor/intro/scheduler.md' contains a link to 'storage/hwameistor/hwameistor/install/deploy.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/ceph-dashboard.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/openebs.md' contains a link to 'storage/images/openebs-1.svg' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/openebs.md' contains a link to 'storage/images/openebs-2.svg' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/openebs.md' contains a link to 'storage/images/openebs-3.svg' which is not found in the documentation files.
WARNING  -  Documentation file 'storage/solutions/openebs.md' contains a link to 'storage/images/openebs-4.svg' which is not found in the documentation files.
WARNING  -  Documentation file 'videos/insight.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
WARNING  -  Documentation file 'videos/insight.md' contains a link to 'insight/quickstart/install-agent.md' which is not found in the documentation files.
```

**Comment by Michelle951** [comment]
local build shows no link errors:
<img width="683" alt="image" src="https://github.com/DaoCloud/DaoCloud-docs/assets/35519562/8818298f-3fa2-48c3-8162-f21a71445cfa">
